tests/nnstreamer_source/unittest_src_iio.cc

```diff
@@ -510,7 +510,7 @@ build_dev_dir_scan_elements (iio_dev_dir_struct * iio_dev,
   num_bytes++;
 
   data_size = num_bytes * iio_dev->num_scan_elements / skip;
-  scan_el_data = (char *) malloc (data_size);
+  scan_el_data = (char *) g_malloc0 (data_size);
```

Comment:
There is another malloc-g_free pair at line 586 (copied_scan_el_data).

Reply:
Added.